### DISCUSSION TITLE
BIRT 4.14.0 - Parameters (String/Date) without "Is Required" are not updated when the report is executed

### DISCUSSION THREAD

**Discussion by Sachinjuneja9090** [eclipse-birt/birt#1959] [Q&A] [answered]
In BIRT 4.14.0, there's an issue where String and Date parameters that are not marked as "Is Required" are not updated properly when the report is run. This means that if a parameter is left blank or has a default value, and it's not set as "Required," BIRT doesn't update or apply the new value when the report is executed.

This problem didn’t occur in BIRT 4.8, where these parameters worked correctly.

**Comment by speckyspooky** [comment, marked as answer]
Please don't open doubled question like issue and discussion. Please see for my questions at your issue ticket #1960
